### ISSUE TITLE
KeyError: '2021_09_09_13_20_58'

### ISSUE THREAD

**Issue by Chinese-Coding** [yifanlu0227/HEAL#18]
# KeyError: '2021_09_09_13_20_58'

## Chinese 
在我训练模型时, 遇到了这样一个错误 (Figure 1), 经过我的调试我发现错误原因为: 在数据集 `OPV2V` 和 `OPV2V-H` train文件夹下有时间戳 `2021_09_09_13_20_58`的数据, 而在 `opv2v_4modality.json` 和 `opv2v_4modality_in_order.json` 中均没有该数据.
一个简单的解决办法就是将它们从对应的目录下移除即可, 或者还有一个方法就是在 `opv2v_4modality.json` 和 `opv2v_4modality_in_order.json` 加上该时间戳的数据, 但是我认为这一步可能会引起训练精度的问题, 所有没有这么做.

## English
When I was training the model, I encountered such an error (Figure 1). After debugging, I found that the cause of the error was: there was data with the timestamp `2021_09_09_13_20_58` in the train folder of the datasets `OPV2V` and `OPV2V-H`, but there was no such data in `opv2v_4modality.json` and `opv2v_4modality_in_order.json`.
A simple solution is to remove them from the corresponding directories, or another way is to add the data with the timestamp to `opv2v_4modality.json` and `opv2v_4modality_in_order.json`, but I think this step may cause problems with training accuracy, so I didn't do it.

## Figure
![image](https://github.com/yifanlu0227/HEAL/assets/137599793/4b083408-bfe3-4693-948f-d3564f2047e7)

![image](https://github.com/yifanlu0227/HEAL/assets/137599793/7766c9ac-4da3-44a2-b53f-30cdbf80734a)

**Comment by Chinese-Coding** [comment]
# Additional findings

## Chinese
我在 `v2xset_4modality.json` 中发现了 `2021_09_09_13_20_58` 的数据, 但是应该是不能使用的, 因为 id 对不上.

## English
I found the data of `2021_09_09_13_20_58` in `v2xset_4modality.json`, but it should not be used because the id does not match.

## Figure
![image](https://github.com/yifanlu0227/HEAL/assets/137599793/a6811a9a-053f-4d9d-83d8-a939d24de325)

![image](https://github.com/yifanlu0227/HEAL/assets/137599793/bf8dee45-040a-4b8b-8498-442967219819)

**Comment by yifanlu0227** [comment]
奇怪，我下载的数据集里`2021_09_09_13_20_58`是V2XSet内的。而且这个Scene因为camera 数据不全，所以我在[这里](https://github.com/yifanlu0227/HEAL/blob/4882bd02514725fa5e2e5717d410de739f33790e/opencood/data_utils/datasets/basedataset/v2xset_basedataset.py#L12) 特地过滤了。

**Comment by Chinese-Coding** [comment]
对, 我也很疑惑这个问题. 我一开始以为是我OPV2V的数据集下载出现了错误, 然而我今天看到您新构建的数据集中也有这个文件, 所以感到困惑. 也许是数据集做了对应的更新, 不过移除之后就好了. 感谢您及时的回复. 🙏 🙏🙏

Yes, I am also confused about this problem. I thought at first that there was an error in downloading my OPV2V dataset, but today I saw that this file also exists in your newly constructed dataset, so I was confused. Maybe the dataset has been updated accordingly, but it will be fine after it is removed. Thank you for your timely reply. 🙏🙏🙏
